### ISSUE TITLE
Accessibility: Anchor color fails WCAG color contrast minimums in dark mode

### ISSUE THREAD

**Issue by calebfoss** [processing/p5.js-website#688]
### Most appropriate sections of the p5.js website?

Other (specify if possible)

### What is your operating system?

None

### Actual Behavior

![Paragraph of text with hyperlinks that are a magenta color with very little contrast against the black background.](https://github.com/user-attachments/assets/26cad467-cff4-4fa2-a440-b10cb63fd4e5)

In dark mode, many anchor elements throughout the site use a magenta color, which has a [2.98:1 contrast ratio](https://webaim.org/resources/contrastchecker/?fcolor=B20046&bcolor=000000) against the black background. This fails all WCAG standards for color contrast.

### Expected Behavior

Colors selected that meet accessible contrast minimums.

### Would you like to work on the issue?

No

**Comment by davepagurek** [comment]
On the tool @calebfoss linked, when I drag the lightness slider up until all tests pass, the color is `#FF579A`, which looks like this:

![Image](https://github.com/user-attachments/assets/dd29290a-65a5-4b1b-bdd5-3ea3ddb3fbe1)

Would we want to switch to that in dark mode, or is there a different brighter color we can be using?

**Comment by ksen0** [comment]
Hi @davepagurek , looks like the color that is intended to be used in dark mode, `--type-magenta`  `#eb285f` and it does pass, but `--type-magenta-dark` is used, which fails: https://github.com/processing/p5.js-website/blob/6d683a8a77d8d1866effe5ae501e4b5dd566e69d/styles/variables.scss#L28 so it's a bug on switching link color in Dark/light.

That color would fail against a white background though, so `eb285f` alone can't be used for both even though it is ok on darkmode.
